.github/workflows/release.yml

```diff
@@ -0,0 +1,86 @@
+# This workflow will tag, release, publish to npm, and upload to gh pages.
+name: Production Release
+
+# workflow will only run when a pull request to the release branch is closed.
+on:
+    push:
+        branches: [release]
+
+jobs:
+    release_master:
+        name: Tag, Release and Publish
+        # only run if PR is merged (not just closed)
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+
+            # we take the message of the commit to use for tag and release body.
+            - name: get commit message
+              run: |
+                  echo ::set-env name=commitmsg::$(git log --format=%b -n 1 ${{ github.event.after }})
+
+            # the tag will match the package.json version (eg. v1.0.0)
+            - name: Tag
+              id: autotagger
+              uses: butlerlogic/action-autotag@stable
+              with:
+                  GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+                  strategy: package
+                  prefix: v
+                  # use the body of the PR commit as the tag message
+                  tag_message: $commitmsg
+
+            - name: Release
+              id: create_release
+              if: steps.autotagger.outputs.tagname != ''
+              uses: actions/create-release@v1.0.0
+              env:
+                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+              with:
+                  tag_name: ${{ steps.autotagger.outputs.tagname }}
+                  release_name: ${{ steps.autotagger.outputs.tagname }}
+                  # use the body of the PR commit as the release body
+                  body: $commitmsg
+                  draft: false
+                  prerelease: false
+
+    publish_to_npm:
```

Comment:
Do you know if it will need permissions to publish inside the @nypl org?

Reply:
Yep, we need to add a secret to the repo called `NPM_TOKEN`